### PR TITLE
Added Read only property to Direct LUN

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

File: utils/mgmt_system/__init__.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def connect_direct_lun_to_appliance(self, vm_name, disconnect):
             disk.set_format("raw")
             disk.set_lun_storage(storage)
             disk.set_shareable(True)
+            disk.set_read_only(True)
```
Comment (psav) [inline utils/mgmt_system/__init__.py]: Well, do we _always_ want it read-only? I thought the idea was to add a flag to the function prototype so we could specify if we _Wanted_ RO behavior or not.

**Comment by dajoRH** [comment]
Lint report for commit e84cdb7bc55b983c33184e66b1f5d05c03698b0e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0c702d77dbe3f54f6fbfd9f3cfe0961456f445fa:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./utils/mgmt_system/__init__.py:24:81: E202 whitespace before ')'
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7dfefd1d9d5e760c66b3510246b63a22f849efab:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: .gitignore [hunk 2]
```diff
@@ -65,4 +65,5 @@ notebooks/.ipynb_checkpoints
 .jenkins*
 cfme-qe-yamls
 rhci-qe-data
+artifacts/artifactor.log
```
Comment (psav) [inline .gitignore]: I think this is specific to your setup, my artifactor logs into $ROOT/log so this is not relevant for everyone, can we please remove it.

**Comment by psav** [comment]
Once the .gitignore change is removed, I'm happy to merge.

**Comment by dajohnso** [comment]
Hey Thom, I never included that when I first wrote the script because it doesn't seem to work. By that, I mean the rhev UI does not show R/O icon on the disk, at least in rhevm34.  We need to recover iscsi storage on rhev35 to verify its the same but I don;t think we can merge this until we see it actually set it to read only.

**Comment by thom-at-redhat** [comment]
Dave, you are right.  In versions < 3.5, hooks are used.  For version >= 3.5, R/O should work and we are going to try nested.

**Comment by psav** [comment]
So @thom-at-redhat are we going to put a version check in there before setting the RO? and just log a warning if RO is not available?

File: utils/mgmt_system/__init__.py [hunk 3]
```diff
@@ -74,6 +74,7 @@ def connect_direct_lun_to_appliance(self, vm_name, disconnect):
             disk.set_format("raw")
             disk.set_lun_storage(storage)
             disk.set_shareable(True)
+            disk.set_read_only(readonly)
```
Comment (psav) [inline utils/mgmt_system/__init__.py]: I think given the info we received, we can put a check here for the provider version, which should be accessible through the API.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mfalesni** [comment]
(sorry for the noise, accidentally clicked a milestone :smile: )

**Comment by jkrocil** [comment]
Closing since mgmt_system is no longer a part of this repo, it has been moved to https://github.com/ManageIQ/wrapanapi.

**Pull request closed without merging**